CHANGELOG.rst

```diff
@@ -14,6 +14,11 @@ Change Log
 Unreleased
 ----------
 
+[0.50.2] - 2017-10-04
+---------------------
+
+* Added UTM parameters to marketing,track_selection and course/programs enrollment URLs returned by Enterprise API.
```

Comment:
`Added UTM parameters to marketing, track selection, and course/program enrollment URLs returned by Enterprise API.`